script/.copyrigth_aliases.json

```diff
@@ -0,0 +1,133 @@
+[
+  {
```

Comment:
Should this file be onder `script`? Feels a bit weird.

Reply:
I moved it in the base dire like before.